packages/components/src/internal/components/search/SearchPanel.spec.tsx

```diff
@@ -8,25 +8,27 @@ import { PaginationButtons } from '../buttons/PaginationButtons';
 
 import entitiesJSON from '../../../test/data/searchResults.json';
 
-import { SearchPanelImpl } from './SearchPanel';
+import { SearchPanelImpl, SearchPanelImplProps } from './SearchPanel';
 
 import { SearchResultCard } from './SearchResultCard';
 
 import { SearchResultsModel } from './models';
 import { getProcessedSearchHits } from './actions';
 
-const DEFAULT_PROPS = {
-    appName: 'test',
-    offset: undefined,
-    onPageChange: jest.fn(),
-    search: jest.fn(),
-    searchResultsModel: undefined,
-    searchTerm: undefined,
-};
+describe('SearchPanelImpl', () => {
+    function defaultProps(): SearchPanelImplProps {
```

Comment:
I tend to use a function so that it news up the props. It makes it easier for things like `jest.fn()` when the mock is used so that is fresh with each test usage.